### ISSUE TITLE
Repetition in vocabulary section

### ISSUE THREAD

**Issue by davebrowning** [w3c/dxwg#704]
In rewiring the internal document links, I noticed we've been a little inconsistent within Ch 6 with repetition of the inherited properties for `dcat:Catalog` from its super-classes `dcat:Resource` and `dcat:Dataset` which we don't repeat in the parallel sections for `dcat:Dataset` and `dcat:DataService`.  It would be easy to remove the duplicates most of the definitions/use e.g. `title` , `description` etc are identical though there are some usage notes that may require rewording to make sense such the various forms of the `language` property.

I think this is worth doing  but wanted to ask if it considered and ruled out?  (As a reader, duplicating in some places and not others without a visible rationale makes me worried I've missed something).

**Comment by agbeltran** [comment]
I expressed my view on this in the new issue I created (#718) having not seen this before. I think we have to remove the descriptions of the inherited properties to be consistent across the document. For the case of ```language```, I would move the extra text when relevant to ```dcat:Resource```. For example, it makes sense to add ```This refers to the language used in the textual metadata describing titles, descriptions, etc. of the resources (i.e. datasets and services) in the catalog.``` in the ```dcat:Resource``` bit (adpating the text when necessary). 

I suggest addressing this after #717 has been merged, as it adds a few more properties to ```dcat:Resource```.

**Comment by davebrowning** [comment]
Let me draft something and we can see if it passes muster - `dcat:Resource` currently reflects that `language` is the language of the data in the `dcat:Dataset`  (or at least that's how I read the usage note [here](https://w3c.github.io/dxwg/dcat/#Property:resource_language)) so some similar text would apply to the data in the catalog (which just happens to be the textual metadata of the resources).  Then there is services to consider...

At a tangent: should `dct:spatial` be moved to `dcat:Resource` as well?  The meaning for `dcat:Catalog` seems the same as for `dcat:Dataset` - its some spatial/geographical characteristic of the contents in both cases?  @dr-shorthair - was this considered or is it just a glitch?  Perhaps its a stretch to apply this to services.

**Comment by davebrowning** [comment]
Actually, strike that last question - `dcat:Catalog` can inherit `dct:spatial` from `dcat:Dataset`.  `dcat:Resource` doesn't need it
